### ISSUE TITLE
Not clear how to disable one check

### ISSUE THREAD

**Issue by korjavin** [mgechev/revive#144]
I would like to have a simple method to disable one check. 
Let's say "exported"
Ideally, with command-arg, but toml config is okay too.

I can't figure out how to do so from the docs.

**Comment by mgechev** [comment]
A rule is enabled only if it exists in your toml config. By default revive runs with the golint configuration. To disable exported you need to create the following toml config file:

```toml
ignoreGeneratedHeader = false
severity = "warning"
confidence = 0.8
errorCode = 0
warningCode = 0

[rule.blank-imports]
[rule.context-as-argument]
[rule.context-keys-type]
[rule.dot-imports]
[rule.error-return]
[rule.error-strings]
[rule.error-naming]
[rule.if-return]
[rule.increment-decrement]
[rule.var-naming]
[rule.var-declaration]
[rule.package-comments]
[rule.range]
[rule.receiver-naming]
[rule.time-naming]
[rule.unexported-return]
[rule.indent-error-flow]
[rule.errorf]
```

**Comment by RoestVrijStaal** [comment]
tl;dr the key is to NOT add the rule `[rule.exports]` to **revive.toml**, or comment it out.
